src/topology/algebra/mul_action.lean

```diff
@@ -44,59 +44,80 @@ class has_continuous_smul (M α : Type*) [has_scalar M α]
 
 export has_continuous_smul (continuous_smul)
 
+/-- Class `has_continuous_vadd M α` says that the additive action `(+ᵥ) : M → α → α`
+is continuous in both arguments. We use the same class for all kinds of additive actions,
+including (semi)modules and algebras. -/
+class has_continuous_vadd (M α : Type*) [has_vadd M α]
+  [topological_space M] [topological_space α] : Prop :=
+(continuous_vadd : continuous (λp : M × α, p.1 +ᵥ p.2))
+
+export has_continuous_vadd (continuous_vadd)
+
+attribute [to_additive] has_continuous_smul
+
 variables {M α β : Type*} [topological_space M] [topological_space α]
 
 section has_scalar
 
 variables [has_scalar M α] [has_continuous_smul M α]
 
+@[to_additive]
 lemma filter.tendsto.smul {f : β → M} {g : β → α} {l : filter β} {c : M} {a : α}
   (hf : tendsto f l (𝓝 c)) (hg : tendsto g l (𝓝 a)) :
   tendsto (λ x, f x • g x) l (𝓝 $ c • a) :=
 (continuous_smul.tendsto _).comp (hf.prod_mk_nhds hg)
 
+@[to_additive]
 lemma filter.tendsto.const_smul {f : β → α} {l : filter β} {a : α} (hf : tendsto f l (𝓝 a))
   (c : M) :
   tendsto (λ x, c • f x) l (𝓝 (c • a)) :=
 tendsto_const_nhds.smul hf
 
+@[to_additive]
 lemma filter.tendsto.smul_const {f : β → M} {l : filter β} {c : M}
   (hf : tendsto f l (𝓝 c)) (a : α) :
   tendsto (λ x, (f x) • a) l (𝓝 (c • a)) :=
 hf.smul tendsto_const_nhds
 
 variables [topological_space β] {f : β → M} {g : β → α} {b : β} {s : set β}
 
+@[to_additive]
 lemma continuous_within_at.smul (hf : continuous_within_at f s b)
   (hg : continuous_within_at g s b) :
   continuous_within_at (λ x, f x • g x) s b :=
 hf.smul hg
 
+@[to_additive]
 lemma continuous_within_at.const_smul (hg : continuous_within_at g s b) (c : M) :
   continuous_within_at (λ x, c • g x) s b :=
 hg.const_smul c
 
+@[to_additive]
 lemma continuous_at.smul (hf : continuous_at f b) (hg : continuous_at g b) :
   continuous_at (λ x, f x • g x) b :=
 hf.smul hg
 
+@[to_additive]
 lemma continuous_at.const_smul (hg : continuous_at g b) (c : M) :
   continuous_at (λ x, c • g x) b :=
 hg.const_smul c
 
+@[to_additive]
 lemma continuous_on.smul (hf : continuous_on f s) (hg : continuous_on g s) :
   continuous_on (λ x, f x • g x) s :=
 λ x hx, (hf x hx).smul (hg x hx)
 
+@[to_additive]
 lemma continuous_on.const_smul (hg : continuous_on g s) (c : M) :
   continuous_on (λ x, c • g x) s :=
 λ x hx, (hg x hx).const_smul c
 
-@[continuity]
+@[to_additive, continuity]
```

Comment:
If you swap the order of the attributes then the additive version of the lemma should get the continuity attribute as well.

Reply:
```suggestion
@[continuity, to_additive]
```